src/sections/intro.mdx

```diff
@@ -0,0 +1,6 @@
+The 2019 Design Systems Survey is intended to illuminate how design systems are created, used, and maintained. Design Systems have the ability to solve real-life problems, establishing consistency in design and code, promoting efficiency when creating products, communicating usage guidance, encouraging reuse instead of rework, increasing the accessibility and usability of digital properties, and so much more.
+
+For the [second year in a row][1], [Sparkbox][2], a web design and development firm, has conducted this survey to learn more about both the challenges and benefits of building and using design systems. By analyzing the reflections of in-house teams who use a design system and agencies who’ve helped organizations build new or maintain existing design systems, we look to inform those who want to build or improve their own.
```

Comment:
Just a note that we'll need to update the links to 2018 survey towards the end of the project since they'll live under `/2018` instead of `designsystemsurvey`.